patcher/src/components/Sound/index.tsx

```diff
@@ -39,40 +40,52 @@ const sounds = {
 export class Sound extends React.Component<SoundProps, SoundState> {
 
   private bgRef: HTMLAudioElement = null;
+  private evhs: any[] = [];       // event handlers
```

Comment:
I don't like the use of any here, but it only used internally in this component, so I can live with it.